samples/dashboard/demos/climate/demo.css

```diff
@@ -64,12 +67,19 @@
     stroke: #986;
 }
 
-#world-map .highcharts-series-1 .highcharts-data-label text {
+#world-map .highcharts-series-1 .highcharts-data-label:nth-child(odd) text {
```

Comment:
We can use default style for text and use only `even` as extension. We do not need to declare odd and even.